packages/core/src/utils/cookie.test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from 'vitest'
+
+import { parseCookie } from './cookie.js'
+
+test('parseCookie', () => {
```

Comment:
No need to update now, but [`test.each`](https://vitest.dev/api/#test-each) could be good for something like this in the future.

Reply:
Did you meant I should remove this test file for this PR?